app/models/recommendation.py

```diff
@@ -98,16 +98,17 @@ async def get_recommendations_from_experiment(
 
         # apply rankers from the slate experiment on the candidate set's candidates
         for ranker in experiment.rankers:
-            if ranker == 'thompson-sampling':
+            if ranker in THOMPSON_SAMPLING_RANKERS:
                 # thompson sampling takes two specific arguments so it needs to be handled differently
-                recommendations = await RecommendationModel.__thompson_sample(slate_id, recommendations)
+                recommendations = await RecommendationModel.__thompson_sample(ranker, slate_id, recommendations)
                 continue
             recommendations = get_ranker(ranker)(recommendations)
```

Comment:
Doesn't have to be in this PR, but I'd like to apply the same pattern here that's used for ranking slates:
```
if ranker in THOMPSON_SAMPLING_RANKERS:
    ranker_kwargs = {
        'metrics': await SlateMetricsFactory(dynamodb_endpoint=dynamodb["endpoint_url"]).get(
            slate_lineup_id,
            [s.id for s in slate_configs])
    }

slate_configs = get_ranker(ranker)(slate_configs, **ranker_kwargs)
```
I think that simplifies the code because there's only a single call to the ranker.